---a/net/minecraft/item/ItemWritableBook.java
+++b/net/minecraft/item/ItemWritableBook.java
@@ -6,47 +6,37 @@
 import net.minecraft.stats.StatList;
 import net.minecraft.world.World;
 
-public class ItemWritableBook extends Item
-{
+public class ItemWritableBook extends Item {
+
     private static final String __OBFID = "CL_00000076";
 
-    public ItemWritableBook()
-    {
+    public ItemWritableBook() {
         this.setMaxStackSize(1);
     }
 
-    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn)
-    {
+    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn) {
         playerIn.displayGUIBook(itemStackIn);
         playerIn.triggerAchievement(StatList.objectUseStats[Item.getIdFromItem(this)]);
         return itemStackIn;
     }
 
-    public static boolean isNBTValid(NBTTagCompound nbt)
-    {
-        if (nbt == null)
-        {
+    public static boolean isNBTValid(NBTTagCompound nbt) {
+        if (nbt == null) {
             return false;
-        }
-        else if (!nbt.hasKey("pages", 9))
-        {
+        } else if (!nbt.hasKey("pages", 9)) {
             return false;
-        }
-        else
-        {
+        } else {
             NBTTagList nbttaglist = nbt.getTagList("pages", 8);
 
-            for (int i = 0; i < nbttaglist.tagCount(); ++i)
-            {
+            for (int i = 0; i < nbttaglist.tagCount(); ++i) {
                 String s = nbttaglist.getStringTagAt(i);
 
-                if (s == null)
-                {
+                if (s == null) {
                     return false;
                 }
 
-                if (s.length() > 32767)
-                {
+                if (s.length() > 32767) {
                     return false;
                 }
             }
@@ -54,4 +54,4 @@
             return true;
         }
     }
-}
+}
